app/config/storage-config.js

```diff
@@ -0,0 +1,33 @@
+const Joi = require('joi')
+
+// Define config schema
+const schema = Joi.object({
+  connectionStr: Joi.string().when('useConnectionStr', { is: true, then: Joi.required(), otherwise: Joi.allow('').optional() }),
+  storageAccount: Joi.string().required(),
+  outboundContainer: Joi.string().default('ffc-sfi-payment-submission-outbound'),
+  archiveContainer: Joi.string().default('ffc-sfi-payment-submission-archive'),
```

Comment:
At the moment you are just using the outboundContainer in the code. Are you planning on having an archive and quarantine container too? If not there is config here (and in the Helm chart) you don't need so should be removed.

Reply:
removed